src/main/java/com/buddy/api/domains/adoption/entities/AdoptionPostFollowUpEntity.java

```diff
@@ -0,0 +1,82 @@
+package com.buddy.api.domains.adoption.entities;
+
+import com.buddy.api.domains.adoption.enums.AdaptationStatusEnum;
+import com.buddy.api.domains.profile.entities.ProfileEntity;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Data
+@Builder
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Table(name = "adoption_post_follow_up")
+public class AdoptionPostFollowUpEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @EqualsAndHashCode.Include
+    @Column(name = "follow_up_id", nullable = false, unique = true)
+    private UUID followUpId;
```

Comment:
```suggestion
    @Column(name = "adoption_post_follow_up_id", nullable = false, unique = true)
    private UUID adoptionPostFollowUpId;
```

para seguir o mesmo padrão dos outros ids.